### PR TITLE
Even with an API TOKEN, you only get 5000 requests per hour, so it'd …

### LINKED ISSUE #47: Rate limited boom.
Probably should check to see if the `GITHUB_TOKEN` is set, and stop (unless there's a `--force`set)

because 

```
octohatrack -c -n boot2docker/boot2docker
Collecting contributors....
Collecting commentors................Traceback (most recent call last):
  File "octohatrack.py", line 2, in <module>
    octohatrack.main()
  File "/usr/src/app/octohatrack/__init__.py", line 25, in main
    code_commentors = get_code_commentors(repo_name, args.limit)
  File "/usr/src/app/octohatrack/helpers.py", line 44, in get_code_commentors
    users.append(get_user("/repos/%s/pulls/%d" % (repo_name, index)))
  File "/usr/src/app/octohatrack/helpers.py", line 86, in get_user
    entry = get_data(uri)
  File "/usr/src/app/octohatrack/helpers.py", line 55, in get_data
    resp = conn.send("GET", uri)
  File "/usr/src/app/octohatrack/connection.py", line 71, in send
    return parse_response(response)
  File "/usr/src/app/octohatrack/response.py", line 103, in parse_response
    raise ValueError(message)
ValueError: You have run out of GitHub request tokens. Set a GITHUB_TOKEN to increase your limit to 5000/hour. Try again in ~44 minutes. 
```

just looks ew.

### PR DESCRIPTION
**Pull request by SvenDowideit**
…be nice not to go boom

Closes #47

Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

File: octohatrack/__init__.py [hunk 1]
```diff
@@ -17,13 +17,17 @@ def main():
 
   repo_name = args.repo_name
 
-  if not repo_exists(repo_name): 
-    print("Repo does not exist: %s" % repo_name)
+  try:
```
Comment (SvenDowideit) [inline octohatrack/__init__.py]: I'm assuming that this is the only code that makes GH API requests, but its not clear :8ball:

Comment (glasnt) [inline octohatrack/__init__.py]: The try/catch is to capture the API :boom: when the `ValueError` is thrown in [response.py](https://github.com/LABHR/octohatrack/blob/b1a56f6e91dc85e2d72c8fc8a56279503c31c4f4/octohatrack/response.py#L103)

Comment (SvenDowideit) [inline octohatrack/__init__.py]: yes, that's why i put it in :) but its not instantly obvious if this codebase does all the API requests in these 3 calls, and then is (only) doing output of data, or if there are other API requests hidden in the rendering code.

**Comment by glasnt** [comment]
Good idea! It's probably better to warn all the time, since 60 GETs is barely anything

**Pull request merged**